Library/OptimizelyClientType.swift

```diff
@@ -62,6 +63,14 @@ extension OptimizelyClientType {
 
     return variant
   }
+
+  public func isFeatureEnabled(featureKey: String) -> Bool {
```

Comment:
Does this trigger an impression event similar to `activate` for experiments? If so, can we just add a comment above the function?

Reply:
The docs don't say anything explicitly about triggering an impression:

> This method traverses the client's datafile and checks the feature flag for the feature key that you specify. [...] The method then evaluates the feature rollout for a user. The method checks whether the rollout is enabled, whether the user qualifies for the audience targeting, and then randomly assigns either on or off based on the appropriate traffic allocation. If the feature rollout is on for a qualified user, the method returns true.